### ISSUE TITLE
Bisync doesn't sync new files created on my PC

### ISSUE THREAD

**Issue by binu0012** [GedasFX/decky-cloud-save#88]
I'm using Google Drive. 

Let's say game X has file A, B, C, D in its save folder on the steamdeck. 

I back it up to my google drive using the sync button in the plugin. 

The files A, B, C, D appear on my Google drive online in the web app. 

The files A, D, C, D  appear on my Google drive desktop app. 

I launch game X on my PC, the saves are synced, and I continue the game on my PC. 

When I quit the game on my PC, additional save files are created on the PC in game X's save directory called E, F, G

The files A, B, C, D, E, F, G appear on my Google drive desktop app. 

The files A, B, C, D, E, F, G appear on my Google drive online in the web app. 

However, when I click sync on the steamdeck, the files E, F, G do not show up, only files A, B, C, D. The progress I made on the PC isn't showing up. 

However, changes made to A, B, C, D on the PC do appear to sync. It's just the new files E, F, G created on the PC are not synced. 

I doubled checked to make sure it's not Google Drive's problem as all the files A, B, C, D, E, F, G show up in the web app online. 

Thanks.

**Comment by GedasFX** [comment]
> However, when I click sync on the steamdeck, the files E, F, G do not show up, only files A, B, C, D. The progress I made on the PC isn't showing up.

Right.... We are syncing the folder or individual files? Maybe the sync path is not configured properly? In my testing I cannot reproduce this case...
